src/test/java/lambda/part2/exercise/FunctionCombinationExercise.java

```diff
@@ -19,22 +19,14 @@ public void personHasNotEmptyLastNameAndFirstName0() {
         assertEquals(false, validate.test(new Person("a", "", 0)));
     }
 
-    // TODO
     // negate1: (Person -> boolean) -> (Person -> boolean)
-    private Predicate<Person> negate1(Predicate<Person> test) {
-        return p -> {
-            // TODO
-            throw new UnsupportedOperationException();
-        };
+    private Predicate<Person> negate1(Predicate<Person> predicate) {
+        return p -> predicate.negate().test(p);
     }
 
-    // TODO
     // validateFirstNameAndLastName: (Person -> boolean, Person -> boolean) -> (Person -> boolean)
     private Predicate<Person> validateFirstNameAndLastName(Predicate<Person> t1, Predicate<Person> t2) {
-        return p -> {
-            // TODO
-            throw new UnsupportedOperationException();
-        };
+        return p ->  t1.test(p) & t2.test(p);

```

Comment:
Why not `&&`? What is the difference between these operators? Are you sure you need `&` here?